pages/src/rtl-switcher/rtl-switcher.ts

```diff
@@ -0,0 +1,81 @@
+import {ready} from '../../../src/modules/esl-utils/decorators/ready';
+import {attr} from '../../../src/modules/esl-base-element/decorators/attr';
+import {bind} from '../../../src/modules/esl-utils/decorators/bind';
+import {boolAttr, ESLBaseElement} from '../../../src/modules/esl-base-element/core';
+
+import {TraversingQuery} from '../../../src/modules/all';
```

Comment:
`import {TraversingQuery} from '../../../src/modules/esl-traversing-query/core';`
+ move to other imports